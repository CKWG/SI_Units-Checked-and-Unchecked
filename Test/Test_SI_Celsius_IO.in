
     100.0°C
  40.0°C     -30.0°C

  100.0°Cxd  100.0°asdC
     1001.0°C
  -5.0 °C  Data_Error
  181.5°  2012.0°C
  -5.0 C  Data_Error

               18.15°C    2025°C
4.2*°C

