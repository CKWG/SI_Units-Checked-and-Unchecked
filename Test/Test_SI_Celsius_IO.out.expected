100.0°C
40.0°C
-30.0°C
Unit_Error
Unit_Error
1001.0°C
Unit_Error
Data_Error
Unit_Error
2012.0°C
Unit_Error
Data_Error
18.1°C
2025.0°C
Unit_Error

